lib/stripe/plan.ex

```diff
@@ -14,27 +14,15 @@ defmodule Stripe.Plan do
   Stripe API reference: https://stripe.com/docs/api#plan
   """
 
-  alias Stripe.Util
-
   @type t :: %__MODULE__{}
 
   defstruct [
     :id, :amount, :created, :currency, :interval, :interval_count,
     :livemode, :metadata, :name, :statement_descriptor, :trial_period_days
   ]
 
-  @response_mapping %{
-    id: :string,
-    amount: :integer,
-    created: :datetime,
-    currency: :string,
-    interval: :string,
-    interval_count: :integer,
-    livemode: :boolean,
-    metadata: :metadata,
-    name: :string,
-    statement_descriptor: :string,
-    trial_period_days: :integer
+  @relationships %{
+    created: DateTime
   }
```

Comment:
I'm a little uncomfortable with the naming of `relationships` in the cases where we're only specifying `DateTime` for some fields. Can't think of a better name, though, so this is a rather empty critique.